shared/devices/index.js

```diff
@@ -51,15 +37,17 @@ class Devices extends Component {
 }
 
 export default connect(
-  (state: any) => {
+  (state: any, {routeState}) => {
```

Comment:
Some of the smart components are typed with `RouteProps`, as you saw below. If the component is only rendered via a route, it'll always receive these own props. I didn't attempt to type the connect calls that weren't already typed since I thought it was out of scope. Would you like me to change anything here? I'm not sure if I've understood your question. :slightly_smiling_face: